appserver/pom.xml

```diff
@@ -619,8 +619,8 @@
                                 <version>${javax.el.version}</version>
                             </artifact>
                             <jarType>impl</jarType>
-                            <specVersion>3.0</specVersion>
-                            <implVersion>3.0.0</implVersion>
+                            <specVersion>${javax.el-api.version}</specVersion>
```

Comment:
Isn't this the actual spec version, not the "spec artefact build version"?

E.g. spec should remain at 3.0, since that's the latest spec.